src/org/jitsi/impl/neomedia/MediaStreamImpl.java

```diff
@@ -1829,6 +1874,57 @@ public long getRemoteSourceID()
     }
 
     /**
+     * @{inheritDoc}
+     */
+    @Override
+    public void addRemoteTrack(MediaStreamTrack mediaStreamTrack)
+    {
+        if (mediaStreamTrack == null)
+        {
+            return;
+        }
+
+        Set<Long> ssrcs = mediaStreamTrack.getEncodingsBySSRC().keySet();
+
+        if (ssrcs == null || ssrcs.size() == 0)
```

Comment:
1. Standard `Map` implementation do not return `null` from `keySet()` so I find it an unnecessary complication of the source code to check for `null`. If anyone is implementing a `keySet` method that returns `null`, I consider it their own problem.

2. `ssrcs.size() == 0` is more clearly expressed as `ssrcs.isEmpty()` and this fact is reported by (certain) static analysis tools.